taipy/core/_orchestrator/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -43,8 +43,8 @@ def _dispatch(self, job: Job):
         future = self._executor.submit(_TaskFunctionWrapper(job.id, job.task), config_as_string=config_as_string)
 
         self._set_dispatched_processes(job.id, future)  # type: ignore
-        future.add_done_callback(self._release_worker)
         future.add_done_callback(partial(self._update_job_status_from_future, job))
+        future.add_done_callback(self._release_worker)
```

Comment:
❓  Can you elaborate? Why do we need to release the worker *after* updating the job status?